app/views/instrument_price_policies/_amount_row.html.haml

```diff
@@ -4,13 +4,14 @@
   = text_field_tag "price_policy_#{price_group.id}[usage_rate]",
     display_usage_rate(price_group, price_policy), size: 8, class: "#{price_group.master_internal? ? 'master_usage_cost' : ''} usage_rate"
 
-%td
-  = t("price_policies.amount")
-  %br
-  = text_field_tag "price_policy_#{price_group.id}[minimum_cost]",
-    params["price_policy_#{price_group.id}"] ? params["price_policy_#{price_group.id}"][:minimum_cost] : number_to_currency(price_policy.minimum_cost, unit: "", delimiter: ""),
-    size: 8,
-    class: price_group.master_internal? ? "master_minimum_cost" : ""
+- if local_assigns[:minimum_cost] != false
```

Comment:
I wonder if at this point, we should pull this out of `instrument_price_policies` into a more shared location?